lib/src/reporter.dart

```diff
@@ -30,7 +30,11 @@ class Reporter {
   bool color = true;
   bool quiet = false;
 
-  Reporter({bool this.color, bool this.quiet});
+  Reporter({this.color: true, this.quiet: false}) {
```

Comment:
#nit these defaults aren't necessary if we have null-awares below

Reply:
I left them in so that when calling the constructor, the IDE would tell you what the defaults are.